plugins/modules/parted.py

```diff
@@ -559,6 +559,23 @@ def parted_version():
     return major, minor, rev
 
 
+def check_parted_fix():
+    """
+    Determines if parted have option --fix (-f). Versions prior
+    to 3.4.64 don't have it. For more information see:
+    http://savannah.gnu.org/news/?id=10114
+    """
+    global parted_exec  # pylint: disable=global-variable-not-assigned
+
+    parted_major, parted_minor, revision = parted_version()
+    if (parted_major == 3 and parted_minor == 4 and revision >= 64):
+        return True
+    if (parted_major == 3 and parted_minor >= 5) or parted_major > 3:
+        return True
```

Comment:
You can simplify this to:
```suggestion
    if (parted_major, parted_minor, revision) >= (3, 4, 64):
        return True
```
(Tuples are compared lexicographically.)

This also makes it easier to read IMO :)